libc/docs/talks.rst

```diff
@@ -0,0 +1,29 @@
+=====
+Talks
+=====
+----
+2023
+----
+* Math functions in LLVM libc or yet another correctly rounded libm - Tue Ly
+
+  * `video <https://www.youtube.com/watch?v=kBSJqVWNQLY>`__
```

Comment:
@lntue I couldn't find a link to publicly accessible slides; do you have that? Mind adding it to this page, or sharing the link with me and I will add?

Reply:
Does the link https://drive.google.com/file/d/1Gchrc3AXKXAWFI1nk8Y1x8UsoQSp8XjF/view from https://fpbench.org/talks/fptalks23.html work?

Reply:
Yes; https://github.com/llvm/llvm-project/pull/84516